certified-connectors/RegexFlow Regular Expression/apiDefinition.swagger.json

```diff
@@ -0,0 +1,151 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "RegexFlow Regular Expression",
+    "description": "The Regular Expression connector enables the use of regular expression.  The connector takes text and a regex pattern as inputs and returns was the pattern  matched and the matched groups.",
```

Comment:
@imreagocs-regexflow please make sure that everything is grammatically correct and free of typographical errors. There are some extra whitespaces.

More information about English strings can be found in [the documentation](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#create-quality-english-language-strings).

Reply:
fixed